.github/workflows/gpu.yml

```diff
@@ -20,20 +19,33 @@ jobs:
         build_type: [Release]
         compiler: [{c: gcc, cxx: g++}]
         shared_library: ['ON', 'OFF']
-    # TODO enable testing on DSS-WINDOWS
-    runs-on:  [ "DSS-LEVEL_ZERO", "DSS-UBUNTU" ]
+        include:
+          - os-labels: ["DSS-LEVEL_ZERO", "DSS-UBUNTU"]
+            os: 'UBUNTU'
+            compiler: [{c: gcc, cxx: g++}]
+            cmake_toolchain_file: 'OFF'
+            level_zero_provider: 'OFF'
+          - os-labels: ["DSS-LEVEL_ZERO", "DSS-WINDOWS"]
+            os: 'WINDOWS'
+            compiler: [{c: cl, cxx: cl}]
+            cmake_toolchain_file: "C:/Users/Administrator/repo/vpckg/scripts/buildsystems/vpckg.cmake"
+            level_zero_provider: 'ON'
+
+    runs-on: ${{matrix.os}}
```

Comment:
shouldn't this be runs-on: matrix.os-labels?

Reply:
Done